google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -276,7 +276,7 @@ def parse_insert(insert_sql, params):
     if not after_values_sql:
         # Case b)
         insert_sql = sanitize_literals_for_upload(insert_sql)
-        return {"sql_params_list": [(insert_sql, None)]}
+        return {"sql_params_list": [(insert_sql, params)]}
```

Comment:
Looks like the previous author of the DB API didn't notice that INSERT from SELECT will be a case `b)`, and it can include some args. Thus, passing `None` instead of args is not correct.